relay-event-normalization/src/normalize/span/description/sql/mod.rs

```diff
@@ -136,11 +140,17 @@ impl Mode {
 }
 
 fn scrub_queries_inner(db_system: Option<&str>, string: &str) -> (Option<String>, Mode) {
-    if let Ok((queries, ast)) = normalize_parsed_queries(db_system, string) {
+    let string = if string.len() > MAX_DESCRIPTION_LENGTH {
+        Cow::Owned(truncate_string(string.to_owned(), MAX_DESCRIPTION_LENGTH))
+    } else {
+        Cow::Borrowed(string)
+    };
+
+    if let Ok((queries, ast)) = normalize_parsed_queries(db_system, &string) {
```

Comment:
Do we expect this to work on a truncated string, if not should we even attempt it?